static/src/stylesheets/amp/_header.scss

```diff
@@ -1,89 +1,175 @@
-.main-header {
-    background: $guardian-brand;
-    position: relative;
-    height: 68px;
+$mobile-medium: 375px; // Breakpoint for our most common device sizes
+$gutter-small: 29px;
+$gutter-medium: 37px;
+$gutter-large: 55px;
+$veggie-burger-small: 42px;
+$veggie-burger-small: 42px;
+$veggie-burger-medium: 52px;
+
+.header {
+    background-color: $guardian-brand;
+}
 
-    @include mq($until: 390px) {
-        height: 63px;
-    }
+.header__inner {
+    display: flex;
+    justify-content: space-between;
+    flex-wrap: wrap;
+    align-items: flex-start;
+}
 
-    @include mq($until: 360px) {
-        height: 57px;
+.header__logo-wrapper {
+    padding-top: $gs-baseline / 2;
+    padding-bottom: $gs-baseline / 2;
+    margin-left: auto;
+}
+
+.header__logo__svg {
+    // Removes extra spacing
+    display: block;
+    height: auto;
+    padding-right: $gs-gutter / 2;
+
+    // Aspect ratio: 16:3
+    @include mq($until: $mobile-medium) {
+        width: 170px;
+        // height: auto doesn't work in Safari
+        height: calc(3 / 16 * 170px);
     }
 
-    .logo-wrapper {
-        position: absolute;
-        right: $gs-gutter;
-        padding-top: $gs-baseline/3;
-        width: inherit;
-        text-align: right;
+    @include mq($from: $mobile-medium, $until: mobileLandscape) {
+        width: 225px;
+        height: calc(3 / 16 * 225px);
+    }
 
-        @include mq($until: mobileLandscape) {
-            right: $gs-gutter/2;
-        }
+    @include mq(mobileLandscape) {
+        width: 260px;
+        // height: auto doesn't work in Safari
+        height: calc(3 / 16 * 260px);
+        padding: ($gs-baseline / 2) ($gs-gutter / 2) 0;
+        margin-right: $gs-gutter / 2;
+    }
+}
 
-        svg {
-            width: 190px;
-            height: $gs-row-height;
+.header__nav {
+    @include fs-textSans(5);
+    line-height: $gs-baseline * 2.5;
+    width: 100%;
+    overflow: hidden;
+
+    @include mq($until: mobile) {
+        font-size: 16px;
```

Comment:
not sure how much amp uses our sass properly yet, but should this come from a typeography mixin?